src/app/components/topnavigation/topnavigation.component.html

```diff
@@ -0,0 +1,9 @@
+<nav class = "top-navigation">
+  <div class="top-navigation__header-block">
+  <div class="top-navigation__name" *scText="rendering.fields.heading"></div>
+</div>
+  <div class="top-navigation__cart-block">
```

Comment:
maybe rename it little bit different that cart-block? maybe something like icon-block or button-block

Reply:
Changed